### ISSUE TITLE
The 1115 master branch's Emulator Testing's AccessibilityNodeInfoTest failed

### ISSUE THREAD

**Issue by utzcoz** [robolectric/robolectric#9802]
```
android.view.accessibility.AccessibilityNodeInfoTest > directAccessibilityConnection_queryChildCount[emulator-5554 - 14] FAILED 
	expected: -1
	but was : 295
```
